sdcm/nemesis.py

```diff
@@ -2334,14 +2334,14 @@ def disrupt_hot_reloading_internode_certificate(self):
         Scylla has the ability to hot reload SSL certificates.
         This test will create and reload new certificates for the inter node communication.
         '''
-        self._set_current_disruption('HotReloadInternodeCertificate')
+        self._set_current_disruption('SslHotReloadingNemesis')
```

Comment:
i would be more precise here, as this is a "internode certificate" or "server certificate"..
calling simple `SslHotReloadingNemesis` is not precise, as there is also a "client side certificate" that is not being touched here.

Reply:
Fixed :-)